Server/JokesOnYou.Web.Api/Services/Interfaces/ITagService.cs

```diff
@@ -1,18 +1,30 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
-using JokesOnYou.Web.Api.Models.Request;
+using JokesOnYou.Web.Api.Helpers;
 using JokesOnYou.Web.Api.Models;
+using JokesOnYou.Web.Api.Models.Request;
 using JokesOnYou.Web.Api.Models.Response;
 
 namespace JokesOnYou.Web.Api.Services.Interfaces
 {
     public interface ITagService
     {
         Task DeleteTagAsync(Tag tag);
+
         Task<Tag> GetTagAsync(int id);
-        Task<TagReplyDto> CreateTagAsync(TagCreateDto tagCreateDto, string userId);
-        Task<IEnumerable<TagReplyDto>> GetAllTagDtosAsync();
+
+        Task<TagReplyDto>
+        CreateTagAsync(
+            TagCreateDto tagCreateDto, string userId
```

Comment:
Question: Is this a format on save issue? The method does not need to be on three lines, one is enough

Reply:
Yes format on save issue. I can fix it